SourceKitStressTester/Sources/SwiftCWrapper/SwiftCWrapper.swift

```diff
@@ -88,8 +88,10 @@ public struct SwiftCWrapper {
 
     // Determine the list of stress testing operations to perform
     let operations = swiftFiles.flatMap { (file, sizeInBytes) -> [StressTestOperation] in
-      if let fileFilter = ProcessInfo.processInfo.environment["SK_STRESS_FILE_FILTER"], !file.contains(fileFilter) {
-        return []
+      if let fileFilter = ProcessInfo.processInfo.environment["SK_STRESS_FILE_FILTER"] {
+        if !fileFilter.split(separator: ",").contains(where: { file.contains($0) }) {
+          return []
+        }
```

Comment:
Could do the split outside of the flatMap so it's not done over and over, but doesn't *really* matter here.

Reply:
I think splitting a string by `,` is the least of our performance concerns for the stress tester and I’d prefer to keep the entire filtering logic localized to a few lines.

Reply:
It'd only be one more line, not really any less localized :P. But as I said, it doesn't matter... was just something I noticed at first glance.

Reply:
I still prefer it like this 😉